resources/web/wwi/Toolbar.js

```diff
@@ -553,8 +555,6 @@ export default class Toolbar {
       if (rw.isMainWindow) {
         rw.setSize(2 * robotWindowWidth, 2 * robotWindowHeight);
         rw.setPosition(margin, margin);
-        if (this.parentNode?.openMainRobotWindow === true)
-          rw.changeVisibility();
```

Comment:
You removed what was introduced in #5753 right? In this case, we should also remove the corresponding documentation

Reply:
Oh yeah, right! I'll do that